examples/org.eclipse.emfcloud.modelserver.example/src/main/java/org/eclipse/emfcloud/modelserver/example/client/ExampleModelServerClient.java

```diff
@@ -149,6 +155,24 @@ private static void handleGet(final ModelServerClient client, final String[] com
       System.out.println(Json.parse(response.body()).toPrettyString());
    }
 
+   private static void handleGetAll(final ModelServerClient client, final String[] commandAndArgs)
+      throws InterruptedException, ExecutionException, TimeoutException, IOException {
+      String format = commandAndArgs.length > 1 ? commandAndArgs[1] : "json";
+      if (format.contentEquals("json")) {
+         Response<List<Model<String>>> response = client.getAll().join();
+         System.out.println(response.body().size() + " models");
+         for (Model<String> model : response.body()) {
+            System.out.println("- " + model.getModelUri() + ": " + model.getContent());
+         }
+      } else {
+         Response<List<Model<EObject>>> response = client.getAll(format).join();
+         System.out.println(response.body().size() + " models");
+         for (Model<EObject> model : response.body()) {
+            System.out.println("- " + model.getModelUri() + ": " + model.getContent().eClass());
+         }
```

Comment:
Why do we need the distinction here? If the `xmi` model content is the same string output as the `eClass` output in my case.

Reply:
I'm not sure what you mean by the same content but I split this method to showcase the difference between the json case where you get "only" the string and the 'xmi' where you get the actual EObject and thus can access the eClass.

Reply:
Sorry for the confusion, I overlooked the format argument difference here.